app/views/catalog/_basic_search.html.erb

```diff
@@ -23,4 +23,5 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+<%= javascript_include_tag 'commonwealth-vlr-engine/basic_search' %>
```

Comment:
what happens if you move this to the top of the partial?

Reply:
Works in the browser, but the feature spec now fails. Adding the include at the end of the partial has been my typical pattern most of the time, it seems to work well?